docs/pages/Cardinal/API-Reference/evm-support.mdx

```diff
@@ -0,0 +1,31 @@
+# EVM Support
+
+Transactions and Reads can be sent from smart contracts in the EVM. This is by an opt-in basis, so you'll need to pass in the corresponding option when instantiating your transactions and reads to enable EVM support.
+
+## Supported Types
+
+Below are the Go types available to use in struct fields when utilizing the `EVMSupport` options.
+
+- []byte (resolves to `bytes` in ABI encoding)
+- string
+- bool
+- int8 - int64
+- uint8 - uint64
+- github.com/ethereum/go-ethereum/common.Address (resolves to `address` in ABI encoding)
+- *math/big.Int (resolves to a specified int or uint above 64 bits i.e. uint128)
+
+Additionally, slices of all the above types work as well.
+
+import { Callout } from 'nextra/components'
```

Comment:
Can we move this import the top

Reply:
[move import to top](https://github.com/Argus-Labs/world-engine/pull/200/commits/2a47bf0d481954a256601a39d1a43fb773dc35cf)